squad/frontend/templates/404.html

```diff
@@ -0,0 +1,6 @@
+{% extends "squad/base.html" %}
+{% load squad %}
+
+{% block content %}
+<h1>Page not found!<h1>
+{% endblock %}
```

Comment:
If I read the docs properly, it doesn't: https://docs.djangoproject.com/en/1.11/topics/http/views/#the-http404-exception
There was no template when I tested locally so I assume there is no default. Each project needs to create it's own.